bert_e/git_host/github/__init__.py

```diff
@@ -508,17 +507,29 @@ def __str__(self) -> str:
         return self.state
 
 
-class AggregatedCheckRuns(base.AbstractGitHostObject,
-                          base.AbstractBuildStatus):
+class WorkflowRuns(base.AbstractGitHostObject):
+    """
+    Endpoint to have access about workflows runs
+    """
+    GET_URL = "/repos/{owner}/{repo}/actions/runs/{id}"
+    CREATE_URL = "/repos/{owner}/{repo}/actions/runs"
+    LIST_URL = CREATE_URL
+    SCHEMA = schema.WorkflowRun
+    LIST_SCHEMA = schema.WorkflowRuns
+
+
+class AggregatedCheckSuites(base.AbstractGitHostObject,
+                            base.AbstractBuildStatus):
```

Comment:
I noticed that a dependabot check-suite stay in there, can you make sure that we are ignoring it? I don't remember if this was the case with check-runs.